pxr/usdImaging/lib/usdviewq/stageView.py

```diff
@@ -908,7 +908,7 @@ def _getRenderer(self):
         # create the renderer lazily, when we try to do real work with it.
         if not self._renderer:
             if self.isValid():
-                self._renderer = UsdImagingGL.Engine()
+                self._renderer = UsdImagingGL.Engine('/', delegateID='/hydra')
                 self._handleRendererChanged(self.GetCurrentRendererId())
```

Comment:
I love the idea of making this feature available for debugging... but please understand there's a non-trivial performance and memory cost to it, because you are forcing the creation of an entire new hierarchy of SdfPaths for the scene, which is a significant and locky process.  So:
1. I wouldn't want people copying this pattern for every rendering app they write
2. Can we put this behind a command-line debug option?

Reply:
Ouch - I didn't realize there would be a significant performance hit.  I guess I had been hoping that since hydra mostly works on a flattened scene graph, introducing an extra top-level item wouldn't have much of an impact.  I didn't realize that SdfPaths are kept in a global table, like TfTokens... well, good to know, anyway!

Anyway - yes, making it a command line option makes sense.